python/viewer/image_viewer.py

```diff
@@ -283,7 +285,19 @@ def sync_streams(image_pbs_path, pc_pbs_path, lidar_pbs_path):
                     lidar = data['lidar'] if 'lidar' in streams else None
                     yield RenderData(image=image, pc=pc, lidar=lidar)
 
-def to_rgb_image(render_data):
+
+MAX_DOPPLER = 20
+cmap_doppler = ScalarMappable(norm=Normalize(vmin=-MAX_DOPPLER,
+                                             vmax=MAX_DOPPLER),
+                              cmap=plt.get_cmap('RdYlGn'))
+
+MAX_ELEVATION = 7
+cmap_elevation = ScalarMappable(norm=Normalize(vmin=-MAX_ELEVATION,
```

Comment:
Do we care about -7m or you just dont want a skewed colormap?

Reply:
There could be some reflections that could have weird negative height. Since we do not filter them out for now, I think it is better to at least see that they are weird thanks to their color. This is what we choose to do in Zenspot.